packages/base/src/features/OpenUI5Support.ts

```diff
@@ -29,48 +29,73 @@ type OpenUI5CoreConfiguration = {
 	}
 };
 
+type ControlBehavior = {
+	getAnimationMode: () => string,
+}
+
+type Localization = {
+	getLanguage: () => string,
+	getLanguageTag: () => string,
+	getRTL: () => string,
+	getTimezone: () => string,
+}
+
 type LocaleData = {
 	getInstance: (locale: string) => Locale,
 }
 
-type CoreTheming = {
-    getThemeRoot: () => string,
+type Theming = {
+	getThemeRoot: () => string,
+	getTheme: () => string,
+	attachApplied: (callback: () => Promise<void>) => void,
+}
+
+type Formatting = {
+	getCalendarType: () => string,
+	getLegacyDateCalendarCustomizing: () => LegacyDateCalendarCustomizing,
+}
+
+type CalendarUtils = {
+	getWeekConfigurationValues: () => {
+		firstDayOfWeek: number | undefined,
+	},
 }
 
 type Locale = {
 	getFirstDayOfWeek: () => number,
 	_get: () => CLDRData,
 };
 
-type VersionUtil = (version: string) => {
-	compareTo: (version: string) => number,
-};
-
-const getCore = () => {
-	return window.sap?.ui?.getCore?.() as OpenUI5Core;
-};
-
 class OpenUI5Support {
+	static isModularCore() {
+		const version = window.sap.ui!.version as string;
+		const parts = version.split(".");
+		return parts && parts[1] && parseInt(parts[1]) >= 116;
+	}
+
 	static isLoaded() {
-		return !!getCore();
+		return typeof window.sap?.ui?.getCore === "function";
```

Comment:
@loginger does this still fit as a criterion for the new bootstrap?

Reply:
No it will not fit with the new bootstrap as we do not expose the global sap.ui.getCore() anymore. IMO we must do a probing require 'sap.ui.require("sap/ui/core/Core")' instead. If the module is not yet required also check the global. This should be ok for both worlds.